pages/account/manage/milestone/[[...data]].js

```diff
@@ -282,17 +305,17 @@ export default function ManageMilestone({ BASE_URL, milestone }) {
             </div>
           </form>
           <div>
+
             <UserMilestone
               milestone={{
-                title,
-                description,
-                url,
-                icon,
-                date,
-                isGoal,
-                dateFormat,
+                title: milestoneWatch[0],
```

Comment:
Is there a way not to use position `[0]`? Because I can see this creating bugs

Reply:
Don't get what you mean here